smoke-test-pipeline.yml

```diff
@@ -73,6 +77,14 @@ stages:
         # Get exit status
         status=$?
         [ $status -eq 0 ] && echo "Test Done!" || (echo "Test had issues" && exit 1)
+
+
+        # Run the infra test
+        bash ./infra-validations.sh
```

Comment:
I'm thinking we can run the infra-validations in a separate stage or job. See east and west in the image below. Both tests need SPK on the file system. We would need to set the env vars in each stage/job. Thoughts?

![image](https://user-images.githubusercontent.com/3119513/71122358-3b7b4700-2195-11ea-9b79-c7e32ac6781e.png)

Reply:
This makes sense. As long as they're independent of each other, it would be efficient to have them run in parallel.

Reply:
Captured here - https://github.com/microsoft/bedrock/issues/863
Experiencing issues with current bedrock Integration tests in this method. May require refactoring.

Reply:
Great, I can try to make this work when I add the integration tests for `spk infra generate`.